releasenotes/notes/42365.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+
+releaseNotes:
+- |
+  **Added** support for retrieves typed extension configuration for the Envoy in the specified pod by running `istioctl proxy-config ecds`.
```

Comment:
**Added** support for retrieving typed extension configuration from Envoy in the specified pod by running `istioctl proxy-config ecds`.

Reply:
updated